spec/models/user_spec.rb

```diff
@@ -17,6 +17,25 @@
   it { expect(subject).to allow_value('https://example.com').for(:homepage) }
   it { expect(subject).to_not allow_value('example.com').for(:homepage) }
 
+  describe 'immutable github handle validation' do
+    context 'when creating a new user' do
+      let(:new_user) { build(:user, github_handle: 'github_handle') }
+
+      it 'allows creating a user with given github handle' do
+        expect(new_user.valid?).to eq(true)
```

Comment:
Minor thing here you might like. You can instead write:
`expect(new_user).to be_valid`